index.bs

```diff
@@ -394,9 +394,19 @@ spec:url; type:dfn; text:scheme
   <section algorithm="htmlportalelement-setsourceurl">
     To <dfn for="HTMLPortalElement">set the source URL of a <{portal}> element</dfn> |element|, run the following steps:
```

Comment:
"set the source URL" is feeling like a worse name for this algorithm now.

Reply:
I'm not sure what a better name is. It's principally used when the embedder directs the portal BC to navigate by changing the `src` attribute. It is also called on BC connection, mainly because we didn't navigate earlier when `src` was set.

"Possibly navigate" sounds a little vague. Any suggestions?